### DISCUSSION TITLE
What should be considered about unknown metabolic ratios?

### DISCUSSION THREAD

**Discussion by Chelton919** [Open-Systems-Pharmacology/Forum#1600] [Questions & Problems]
Hello everyone.
I know the proportion of enzyme metabolism contribution of a drug and the fraction excreted unchanged in urine. These fractions add up to a value of 80%. But in the software simulation, the sum of the enzyme metabolism contribution ratio and the fraction excreted to urine is 100%. I wonder how the unknown 20% should be considered? Is it possible to convert to 100% by a co-multiplication factor? Or is it possible to add total hepatic clearance and attribute that 20% here?

**Comment by tobiasK2001** [comment]
Dear Chelton,

hm, a bit hard to answer without knowing more details. I would tend to not use total hepatic clearance but instead to add an enzyme in liver and name unspecific process or so. Than it is easier to calculate the massbalance in the simulation again. B.t.w. did you think about that the missing 20 % in your known massbalance might be not absorbed compound in case it was po administration? 
Best, Tobias

**Reply by Chelton919** [reply]
Thank you for your advice, it helps me a lot！ I would like to ask if the unknown 20% of this compound is not absorbed, can the known 80% be normalized to 100% by the ratio？

**Reply by RoachWang** [reply]
Hello, how did you end up solving this problem?
I am experiencing the same problem.
